foundryup-zksync/foundryup-zksync

```diff
@@ -178,6 +178,59 @@ EOF
       fi
     done
 
+    # Begin anvil-zksync installation
+    say "downloading anvil-zksync"
+
+    # Supported targets for anvil-zksync
+    SUPPORTED_TARGETS=(
```

Comment:
I understand that the doc change was unintended... but windows doesn't seem supported anyways

Reply:
`anvil-zksync` does not support windows.